build/ci/golangci.yml

```diff
@@ -31,6 +31,8 @@ linters-settings:
       - ^testhelper.(GetFileContent|Glob)$
       # No "filepath" package, use "internal/pkg/filesystem" package instead
       - ^filepath\..*$
+      # "httpmock" singleton is forbidden, it doesnt't work in parallel tests, use client.Transport = httpmock.NewMockTransport() instead
+      - ^httpmock\.(Activate|ActivateNonDefault|ActivateNonDefault|DeactivateAndReset|RegisterResponder$)$
```

Comment:
Doteraz sa v testoch pouzivala kniznica `httpmock` priamo, napr. `httpmock.Activate()` prepise globalnu premennu HTTP `DefaultTransport` a tak su vsetky requesty mockovane.

To funguje dobre, pokial testy nebezia paralelne (aktualny stav), ... no ak sa spustia testy parallelne, tak menia tento globalny stav naraz, a to vediet ku koliziam.

Riesenim je v kazdom teste vytvorit vlastnu instanciu `MockedTransport`, ktora sa nastavi HTTP klientovi pouzitemu v teste.